portfolio/src/main/webapp/images/Rainbow-Vortex.svg

```diff
@@ -0,0 +1 @@
+﻿<svg xmlns='http://www.w3.org/2000/svg' width='100%' height='100%' viewBox='0 0 1600 800'><rect fill='#38ffc0' width='1600' height='800'/><g stroke='#000' stroke-width='66.7' stroke-opacity='0.05' ><circle fill='#38ffc0' cx='0' cy='0' r='1800'/><circle fill='#00f1c6' cx='0' cy='0' r='1700'/><circle fill='#00e2cb' cx='0' cy='0' r='1600'/><circle fill='#00d3ce' cx='0' cy='0' r='1500'/><circle fill='#00c3ce' cx='0' cy='0' r='1400'/><circle fill='#00b3cd' cx='0' cy='0' r='1300'/><circle fill='#00a4c9' cx='0' cy='0' r='1200'/><circle fill='#0094c2' cx='0' cy='0' r='1100'/><circle fill='#0084b9' cx='0' cy='0' r='1000'/><circle fill='#0074ad' cx='0' cy='0' r='900'/><circle fill='#00659f' cx='0' cy='0' r='800'/><circle fill='#00558f' cx='0' cy='0' r='700'/><circle fill='#06467e' cx='0' cy='0' r='600'/><circle fill='#1f386c' cx='0' cy='0' r='500'/><circle fill='#262a5a' cx='0' cy='0' r='400'/><circle fill='#281c47' cx='0' cy='0' r='300'/><circle fill='#251035' cx='0' cy='0' r='200'/><circle fill='#210024' cx='0' cy='0' r='100'/></g></svg>
```

Comment:
Make sure to include newlines at the end of all your files - see https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file for a little bit on this! Same in your other files.

Reply:
Oh, didn't know it applied to SVGs, done!